book/website/collaboration/remote-collab/remote-collab-code-review.md

```diff
@@ -0,0 +1,34 @@
+# Collaborative Code Review
+
+## 1. Problem statement
+* Ensuring your code to be reviewed is accessible/reproducible (across different operating systems and coding setups)
+* Trying to learn a new language and not knowing the most efficient way to solve a given problem. Others might have better ways to solve that problem which you might not know about. 
+* Getting feedback about our code's interface, in order to make it more usable.
+* How do you get this feedback while being considerate of other people's time?
+* Getting started and gauging interest in a group code review
+
+## 2. Solutions & Recommendations
+* Code with comprehension and (try to)  make the code reviewing process easier by being clear and organised with the format and logic. The code itself does not have to look professional due to limited experience of the coder, but should be equipped with comments that help reviewers to understand. Good format to allow advanced coding (Object-oriented programming, list comprehension) to be applied.  
```

Comment:
Do you have links or references you could include here for object-orientated programming and list comprehension? Even better if they're within the _Turing Way_!